databases/schemas/modlog-schema.sql

```diff
@@ -0,0 +1,26 @@
+ CREATE TABLE IF NOT EXISTS modlog (
+    modlog_id INTEGER NOT NULL PRIMARY KEY,
+    -- UNIX timestamp
+    timestamp INTEGER NOT NULL,
+    --- roomid OR global-roomid
+    roomid TEXT NOT NULL,
+    action TEXT NOT NULL,
+    visual_roomid TEXT,
+    action_taker TEXT,
+    userid TEXT,
+    autoconfirmed_userid TEXT,
+    ip TEXT,
+    note TEXT
+);
+
+CREATE TABLE IF NOT EXISTS alts (
+    alts_id INTEGER NOT NULL,
```

Comment:
`alts_id` should be called `modlog_id`, you'd name a column `alts_id` if it was the integer primary key for `alts`

Reply:
`(alts_id, userid)` is the primary key for this table

Reply:
should establish foreign key between what is now called `alts_id` and `modlog_id`